core/tests/puppeteer-acceptance-tests/spec/exploration-editor-tests/exploration-creator-manager.spec.js

```diff
@@ -0,0 +1,76 @@
+
+// Copyright 2024 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for Exploration Creator and Exploration Manager
+ */
+
+const userFactory = require(
+  '../../puppeteer-testing-utilities/user-factory.js');
+const testConstants = require(
+  '../../puppeteer-testing-utilities/test-constants.js');
+
+const DEFAULT_SPEC_TIMEOUT = testConstants.DEFAULT_SPEC_TIMEOUT;
+
+describe('Exploration Creator and Exploration Manager', function() {
+  let explorationCreator = null;
+  let superAdmin = null;
+
+  beforeAll(async function() {
+    guestUser1 = await userFactory.createNewGuestUser(
+      'guestUsr1', 'guest_user1@example.com');
+    guestUser2 = await userFactory.createNewGuestUser(
+      'guestUsr2', 'guest_user2@example.com');
+    guestUser3 = await userFactory.createNewGuestUser(
+      'guestUsr3', 'guest_user3@example.com');
+    explorationCreator = await userFactory.createExplorationCreator(
+      'explorationAdm');
+    superAdmin = await userFactory.createNewSuperAdmin('Leader');
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should perform exploration creation and basic actions',
+    async function() {
+      await superAdmin.assignRoleToUser(
+        'explorationAdm', 'voiceover admin');
+
+      await explorationCreator.createExploration();
+      await explorationCreator.goToBasicSettingsTab();
+      await explorationCreator.expectTitleToHaveMaxLength(36);
```

Comment:
@seanlip i think this is now resolved .

Reply:
I'm not sure it's resolved. If I understand Sean correctly, the creator doesn't only expect the length to be below 36, they expect the title to be equal to "Your Title here".

Reply:
@seanlip its also amswered , as it was mentioned in doc to check only for title to be below 36 so.. 
![Uploading Screenshot 2024-02-29 at 1.14.30 AM.png…]()

Reply:
This acceptance test must be written from the perspective of an end user. An end user will not go to the exploration editor, update the title, and expect it to have a length of less than 36. They might expect a specific warning to be shown. Or they might expect that, if they try to enter a title that has > 36 chars, it shows an error (or if they enter characters past a certain length, it shows the truncated title only). And so on.

Your test should be written so that it reads similar to end user actions, as described above. If the spreadsheet does not match this then the spreadsheet needs modification and you might want to request clarification for those cases.

Also the link in your last comment doesn't load for me FYI.